src/contractTest/java/uk/gov/hmcts/reform/em/npa/consumer/NpaPactConsumerTest.java

```diff
@@ -168,4 +168,76 @@ void testGetMarkupsByDocumentId(MockServer mockServer) {
                 .body("[0].rectangles[0].x", equalTo(10.5f));
     }
 
+    @Pact(consumer = "em_npa_api", provider = "native_pdf_annotator_api_provider")
+    public V4Pact updateMarkUpPact(PactBuilder builder) {
+        return builder
+                .usingLegacyDsl()
+                .given("A valid RedactionDTO with ID " + REDACTION_ID + " exists")
```

Comment:
We update the state for both Consumer and Provider to "A valid RedactionDTO with ID exists." The path or body used in the builder already ensures this is passed.

Please apply to any previously implemented methods.